LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-autospace/text-autospace-no-001-ref.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
 <style>
-.no-as { text-autospace: no-autospace; }
+div { text-autospace: no-autospace; }
 .manual-spacing {
     margin: 0px 0.125em;
 }
 </style>
-<div class="no-as"><span>国国</span><span class="manual-spacing">AA</span><span>国国</span><span class="manual-spacing">AA</span><span>国国</span></div>
-<div>国国&ZeroWidthSpace;AA&ZeroWidthSpace;国国&ZeroWidthSpace;AA&ZeroWidthSpace;国国</div>
+<div><span>国国</span><span class="manual-spacing">AA</span><span>国国</span><span class="manual-spacing">AA</span><span>国国</span></div>
+<div>国国AA国国AA国国</div>
```

Comment:
I would leave this line alone. It's good for a reference to have some redundancy.